modules/core/shared/src/main/scala/lucuma/core/syntax/Time.scala

```diff
@@ -38,7 +39,10 @@ trait ToInstantOps {
     new InstantOps(i)
 }
 
-object instant extends ToInstantOps
+object instant extends ToInstantOps {
+  extension(i: Instant)
+    def toTimestamp: Option[Timestamp] = Timestamp.fromInstant(i)
```

Comment:
I suppose we should add (or else just always use) a version that truncates to microseconds.  `fromInstantTruncated` or something like that it is called.